### PR TITLE
add geometry analyzer for quick validation

### PR DESCRIPTION
**Pull request by deguio**
#### PR description:
A geometry analyzer is added to help with the validation of the evolving HGC geometry.
- possibility to run different geometries
- summary text file of the transition regions between different sensor types for CEE and CEH
- script to produce plots like the ones [here](http://deguio.web.cern.ch/deguio/hgcal/geometry/geom_output_v10_D41/)

@rovere @bsunanda @cseez @pfs

### CODE REVIEW

**Comment by ianna** [comment]
please test

File: Geometry/HGCalGeometry/test/python/hgcGeomAnalyzer_cfg.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+process = cms.Process("demo",eras.Phase2C8)
+
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+if options.geom == 'v10':
+    process.load('Configuration.Geometry.GeometryExtended2026D41Reco_cff')
+elif options.geom == 'v11':
+    process.load('Configuration.Geometry.GeometryExtended2026D46Reco_cff')
+else:
+    print 'UNKNOWN GEOMETRY!'
```
Comment (kpedro88) [inline Geometry/HGCalGeometry/test/python/hgcGeomAnalyzer_cfg.py]: better to raise an exception here

**Comment by cvuosalo** [comment]
+1

**Comment by kpedro88** [comment]
+upgrade

**Comment by fabiocos** [comment]
+1

the provided test configuration runs smothly and provide output

**Pull request merged**